src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -2548,6 +2548,7 @@ or ErrorCode.ERR_SingleInapplicableBinaryOperator
                 or ErrorCode.ERR_SingleInapplicableUnaryOperator
                 or ErrorCode.ERR_AmbigOperator
                 or ErrorCode.ERR_UnexpectedArgumentListInBaseTypeWithoutParameterList
+                or ErrorCode.ERR_EncUpdateRequiresEmittingExplicitInterfaceImplementationNotSupportedByTheRuntime
```

Comment:
>ERR_EncUpdateRequiresEmittingExplicitInterfaceImplementationNotSupportedByTheRuntime [](http://example.com/codeflow?start=29&length=84)

Isn't  this a build only diagnostic?

Reply:
Other EnC diagnostics also return false here. E.g. https://github.com/tmat/roslyn/blob/90015551d370dbab5be5e20e5cec9c627451267e/src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs#L1651

Reply:
> Not sure if that should be considered build only or not.

I think all emit time diagnostics are build-only diagnostics. Getting a diagnostic for a syntax tree won't return them